src/components/Form/Controls/MFA/MFA.module.css

```diff
@@ -14,26 +15,98 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.mfa {
-  inline-size: fit-content;
+.container {
+  --gap: var(--cpd-space-3x);
+  --digit-size: var(--cpd-space-10x);
+  --digit-height: var(--cpd-space-12x);
+
   display: flex;
+  inline-size: fit-content;
   flex-direction: row;
-  gap: 1rem;
+  gap: var(--gap);
+
+  /* The input is positioned absolutely
+     so the container needs to be positioned relatively */
   position: relative;
 }
 
-.mfa-input {
-  opacity: 0;
+.control {
+  all: unset;
+
+  /* TODO: a font stack for monospace fonts should be defined in the design tokens
+   * Font stack from https://qwtel.com/posts/software/the-monospaced-system-ui-css-font-stack/#the-monospaced-system-ui-css-font-stack */
+  font-family: ui-monospace, Menlo, Monaco, "Cascadia Mono", "Segoe UI Mono",
+    "Roboto Mono", "Oxygen Mono", "Ubuntu Monospace", "Source Code Pro",
+    "Fira Mono", "Droid Sans Mono", "Courier New", monospace;
+  font-weight: bold;
```

Comment:
I think we should follow what Element Web currently does. => using `Inconsolata` and the same font stakc as them

Reply:
Should I just copy the font stack or wait for it to be formalised in the design tokens?

Reply:
I think it deems to be part of the design tokens in the long run